site/posts/tutorial-seminar-series.ipynb

```diff
@@ -0,0 +1,58 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because we are unable to host an in-person tutorial for this fall season, the Xdev team has decided to switch to a virtual seminar series. We will be hosting one hour tutorial sessions scheduled for the 2nd and 4th Wednesdays of each month at 1 PM starting in October. The first tutorial session, covering how to create your first Python script, will be on October 14th. Some tutorials will be 1-off seminars on specific topics, while others will span multiple seminar sessions.\n",
+    "\n",
+    "Remote tutorials offer their own challenges; no one from our team can walk around the room to notice if anyone needs technical help or one-on-one assistance during a challenging topic. We believe that one hour sessions make it easier for participants to reach out and get the help they need to catch up before the next session of a multi-part seminar, so that we can more effectively reach our education goals.\n",
+    "\n",
+    "## Calendar\n",
+    "All events are at 1 PM MT\n",
+    "\n",
+    "**October 14** --  “Your First Python Script - Reading in a .txt File\"\n",
```

Comment:
I think it would useful to have a brief description with more details about each session so as to help the attendees determine whether the session is the right fit... Also, would it be useful to link to/reference the existing material residing on https://ncar.github.io/python-tutorial/tutorials/beginner.html ?

Reply:
These are good ideas, I didn't want the post to be too long, but I think it covers these bases now.